narwhal/config/tests/snapshots/config_tests__committee.snap

```diff
@@ -1,28 +1,28 @@
 ---
-source: config/tests/config_tests.rs
+source: narwhal/config/tests/config_tests.rs
 expression: committee
 ---
 {
   "authorities": {
-    "i9DpjC/kZbDd57csSg7qWFMF3uTECO2jBD3M5BOXU8T7AebTrMO+9D5IbnUI8N7lFhmjEiKFhwUOlFGbx1jkdP212PR/XvnayHxDaJuMkytxACBnshi9TrbZiZP10tqL": {
+    "ku8PylNwBUMzkshpDSeMnbV25O8rPkUFDPcqAyeEXztH02eOq6WemzFMEsi1p0LrBvKBvTLpwLwHhkCatYptihcO75iZRjtFkYwkAlv0GmQQyHZ+xO+UYweLQiYdFHtf": {
```

Comment:
turns out this Rng change caused the config to change. it's been a while for me to remember, does this have any downstream impact on narwhal config other than updating this snapshot? cc @mwtian

Reply:
I think nothing else needs to happen. Even if a genesis is needed, it should happen as part of the release process.